packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -662,6 +662,7 @@ export const FooterChildStruct = selectiveUnion((value) => {
  */
 export const FooterStruct: Describe<FooterElement> = element('Footer', {
   children: FooterChildStruct,
+  requireScroll: optional(boolean()),
```

Comment:
Please add a test for this in `validation.test.tsx`

Reply:
Will do, good catch!